src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -100,6 +100,37 @@ public AcquireTokenForClientParameterBuilder WithAzureRegion(bool autoDetectRegi
             return this;
         }
 
+        /// <summary>
+        /// Specifies if the token request should be sent to regional ESTS.
+        /// If set, MSAL tries to auto-detect and use a regional Azure authority. This helps keep the authentication traffic inside the Azure region. 
+        /// If the region cannot be determined (e.g. not running on Azure), MSALClientException is thrown with error code region_discovery_failed.
+        /// This feature requires configuration at tenant level.
+        /// By default the value for this variable is false.
+        /// See https://aka.ms/msal-net-region-discovery for more details.
+        /// </summary>
+        /// <param name="autoDetectRegion"><c>true</c> if the token request should be sent to regional ESTS. The default is <c>false</c>.
+        /// </param>
+        /// <param name="fallbackToGlobal"><c>true</c> to fallback to global ESTS endpoint when calls to regional ESTS fail.</param>
```

Comment:
>when calls to regional ESTS fail [](start = 91, length = 32)

this should only happen when we are not able to detect a region or if there is no provided region